index.js

```diff
@@ -63,13 +63,14 @@
           evt.stopPropagation();
           var source = evt.target;
           var found = false;
+          var ignoreClass = this.props.clickOutsideIgnoreClass || IGNORE_CLASS;
```

Comment:
This probably needs to be mentioned in the README.md, too, before we land this, so people know they can use it. It might also be better to call this `outsideClickIgnoreClass` or `onClickOutsideIgnoreClass` (probably the first?)